website/src/data/components/network/props.js

```diff
@@ -137,7 +137,7 @@ const props = [
     {
         key: 'nodeBorderWidth',
         group: 'Nodes',
-        type: 'number | (node: Node) => number',
+        type: 'Function',
```

Comment:
It can be either a number or a function, a function is always used internally for consistency, but you can use a static number as in the website demo for example.